### PR TITLE
Fix deleteOrphaned on mysql. Issue #181

### PR DESCRIPTION
**Pull request by matiasdelellis**
It's ugly, but I know it works in mariadb/mysql :disappointed:

### CODE REVIEW

**Comment by matiasdelellis** [comment]
Well, I received an email from Travis that it had failed when it had not yet started. :sweat_smile: 
But finally it pass correctly.

https://travis-ci.org/matiasdelellis/facerecognition/builds/615052529?utm_medium=notification&utm_source=email

File: lib/Db/PersonMapper.php [hunk 1]
```diff
@@ -297,16 +297,26 @@ public function removeIfEmpty(int $personId) {
 	 */
 	public function deleteOrphaned(string $userId): int {
 		$sub = $this->db->getQueryBuilder();
-		$sub->select(new Literal('1'));
+		$sub->select('*');
```
Comment (stalker314314) [inline lib/Db/PersonMapper.php]: any reason why you needed "*" here? you should keep "1" unless there is reason not to.

**Comment by stalker314314** [comment]
I commented couple of days ago "https://github.com/matiasdelellis/facerecognition/pull/177#issuecomment-555235688_", like we lost Travis. This is probably why I didn't see build failure, Sorry for creating problem.

BTW, how is this working for FaceMapper::deleteUserFaces which uses similar construct:
https://github.com/matiasdelellis/facerecognition/blob/master/lib/Db/FaceMapper.php#L165

I vaguely remember we already had similar issues (but with SQLite...). Anyway, I approved this, shoot when ready!:)

File: lib/Db/PersonMapper.php (hunk 1 above)
Comment (matiasdelellis) [reply lib/Db/PersonMapper.php]: No reason ... it was just one of many tests. :sweat_smile:

**Comment by matiasdelellis** [comment]
> BTW, how is this working for FaceMapper::deleteUserFaces which uses similar construct:
https://github.com/matiasdelellis/facerecognition/blob/master/lib/Db/FaceMapper.php#L165

I don't know.. . As I am understanding now mysql has conflicts with some nested queries, and doctrine with NOT EXISTS in some versions..
That's why I take what was working, and did the delete separately. It was the only thing that worked for me.. But I can't defend this change ..

**Comment by matiasdelellis** [comment]
Well, with last commit pass again.. So, merge it.. Then we can improve it ..

**Pull request merged**